CHANGELOG.md

```diff
@@ -1,3 +1,81 @@
+### citus v9.3.0 (May 6, 2020) ###
```

Comment:
I'd re-order things a bit, put more useful things to many users goes top such as

so, `Adds support for window functions on coordinator` should come first and continues with
`Improves shard pruning logic to understand OR-conditions`
`Prevents using an extra connection for intermediate result multi-casts`
`Adds propagation of  `ALTER ROLE .. SET` statements`
`Introduces a UDF to truncate local data after distributing a table`

and all crash fixes goes down. Also, it'd be nice to have all local execution related changes to be next to each other in the list.

Also, we seem to miss description on https://github.com/citusdata/citus/pull/3692, now I added. Can you get it  here? It should also be on the top block